### DISCUSSION TITLE
After installation, can anyone tell me if this is a normal interface?

### DISCUSSION THREAD

**Discussion by xxbetta** [LycheeOrg/Lychee#2523] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/afbe11e6-e6b2-4c65-8bde-4ca09d163ee4)

ver:5.5.1
php8.2

**Comment by ildyria** [comment, marked as answer]
Yes.
Click on top left and enter username & password to login.

**Reply by xxbetta** [reply]
But there is no response when clicked, and there are no error prompts. I don't know where I went wrong?

**Reply by ildyria** [reply]
Can you open your console and check for any error messages?
Also to make it simpler, I would recommend to set `LIVEWIRE_ENABLED=false` in your `.env` file.

**Reply by xxbetta** [reply]
Do I need to enable this option LIVEWIRE-INABLED=true? After enabling, view the message log and provide the information to you? If so, can you tell me how to obtain the message log after activation and provide it to you.

**Comment by xxbetta** [comment]
Do I need to enable this option LIVEWIRE-INABLED=true? After enabling, view the message log and provide the information to you? If so, can you tell me how to obtain the message log after activation and provide it to you.
________________________________
发件人: Beno?t Viguier ***@***.***>
发送时间: 2024年8月27日 19:15
收件人: LycheeOrg/Lychee ***@***.***>
抄送: xxbetta ***@***.***>; Author ***@***.***>
主题: Re: [LycheeOrg/Lychee] After installation, can anyone tell me if this is a normal interface? (Discussion #2523)


Can you open your console and check for any error messages?
Also to make it simpler, I would recommend to set LIVEWIRE_ENABLED=false in your .env file.

―

**Reply by ildyria** [reply]
Nah, that won't be necessary, the Livewire front-end will be removed in v6. I do not care about what errors it currently generates. :)

**Reply by eviawriter** [reply]
Ah, so basically Livewire is going to be deprecated? I actually had to disable it (`LIVEWIRE_ENABLED=false`) to make the whole app work...

**Reply by ildyria** [reply]
@eviawriter See here:
https://lycheeorg.github.io/2024-06-25-performance-problems/
https://lycheeorg.github.io/2024-06-29-future-of-lychee/

**Comment by xxbetta** [comment]
I solved the problem by using LIVEWIRE-INABLED=true. Thank you for your active help and reply. I hope the project can continue to be optimized. Thank you.
________________________________
发件人: Beno?t Viguier ***@***.***>
发送时间: 2024年8月28日 22:28
收件人: LycheeOrg/Lychee ***@***.***>
抄送: xxbetta ***@***.***>; Author ***@***.***>
主题: Re: [LycheeOrg/Lychee] After installation, can anyone tell me if this is a normal interface? (Discussion #2523)


Nah, that won't be necessary, the Livewire front-end will be removed in v6. I do not care about what errors it currently generates. :)

―
